examples/observability_advance/README.md

```diff
@@ -0,0 +1,3 @@
+# Provision IBM Cloud Monitoring, Log Analysis and Activity Tracker with archiving to an encrypted COS bucket. Additionally, provision Activity Tracker with event routing to COS bucket, Event streams and Log Analysis
```

Comment:
Make the title more readable, maybe `Provision IBM Cloud Monitoring, Log Analysis and Activity Tracker`. The rest of the detail is included in the content.